community/kernel/src/main/java/org/neo4j/kernel/impl/storemigration/monitoring/VisibleMigrationProgressMonitor.java

```diff
@@ -67,7 +68,7 @@ public void completed()
         public void progress( long add )
         {
             current += add;
-            int percent = max == 0 ? 100 : (int) (current*100/max);
+            int percent = max == 0 ? HUNDRED : Math.min( HUNDRED, (int) ((current * HUNDRED) / max) );
```

Comment:
yes it's true, but that will prevent us from having incorrect percentage presented to end user in any cases, it's better to wait a bit longer with 100 percent presented then to see 120 so i would like too keep it.
I can introduce flag that we can use for testing to check for counter overflow, wdyt? did that initially but did not find a lot of use of it.